test/enigma_test.rb

```diff
@@ -13,34 +13,56 @@ def test_it_exists
   end
 
   def test_it_can_generate_a_random_key
-    @enigma.stubs(:rand_key).returns(24689)
-    assert_equal 24689 , @enigma.rand_key
+    @enigma.stubs(:rand_key).returns(02715)
+    assert_equal 02715 , @enigma.rand_key
   end
```

Comment:
changes were made to the testing suite to bring things more in line with the interaction pattern for ease of testing